### DISCUSSION TITLE
Embedding DAGs in the image or using volumes in production

### DISCUSSION THREAD

**Discussion by w7089** [apache/airflow#22776] [Q&A] [answered]
What’s is the best practice for creating/updating/deleting dags in production where Airflow is installed? 

Is it embedding DAGs in airflow image? Or mounting them via volume and updating them on the storage the volume is mounted from?

Are there official docs/demos on this? 

My question is mostly in the context of CI/CD and when there are no clear requirements.

**Comment by potiuk** [comment, marked as answer]
You can see our helm chart https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#managing-dag-files - this is as close as can be.
